<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddBrotherWindow</class>
 <widget class="QDialog" name="AddBrotherWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Brother</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QFrame" name="frame_7">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Personal</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="0">
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QFrame" name="frame_2">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="5" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Last Name</string>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>First Name</string>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Middle Name</string>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="line_f_name"/>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="line_m_name"/>
             </item>
             <item row="5" column="1">
              <widget class="QLineEdit" name="line_l_name"/>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QFrame" name="frame_3">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Phone</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="line_phone"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Email</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="line_email"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Profile</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QFrame" name="frame_4">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_8">
             <item row="0" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Congregation</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="combo_congregation"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Responsibility</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QComboBox" name="combo_publisher">
               <item>
                <property name="text">
                 <string>Elder</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Ministerial Servant</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Publisher</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QFrame" name="frame_5">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_9">
             <item row="0" column="0" colspan="3">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Brother serves in the following capacity:</string>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QCheckBox" name="check_chairman">
               <property name="text">
                <string>Chairman</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QCheckBox" name="check_speaker">
               <property name="text">
                <string>Speaker</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QCheckBox" name="check_talkC">
               <property name="text">
                <string>Talk Coordinator</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>Notes</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QFrame" name="frame_6">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="0" column="0">
              <widget class="QTextEdit" name="text_note"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="check_batch">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Batch</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="button_close">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_add">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>line_f_name</tabstop>
  <tabstop>line_m_name</tabstop>
  <tabstop>line_l_name</tabstop>
  <tabstop>line_phone</tabstop>
  <tabstop>line_email</tabstop>
  <tabstop>combo_congregation</tabstop>
  <tabstop>combo_publisher</tabstop>
  <tabstop>check_chairman</tabstop>
  <tabstop>check_speaker</tabstop>
  <tabstop>check_talkC</tabstop>
  <tabstop>text_note</tabstop>
  <tabstop>button_close</tabstop>
  <tabstop>button_add</tabstop>
  <tabstop>check_batch</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>button_close</sender>
   <signal>clicked()</signal>
   <receiver>AddBrotherWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
